merlin/util/spatialfeature.py

```diff
@@ -635,6 +636,28 @@ def simple_clean_cells(cells: List) -> List:
             if len(cell.get_bounding_box()) == 4 and cell.get_volume() > 0]
 
 
+def get_fov_boxes(fovs: List, alignTask: analysistask.AnalysisTask) -> List:
+    """
+    Creates a list of shapely boxes for each fov containing the global
```

Comment:
This function would make sense within the global align task and it could be renamed to get_fov_bounding_boxes. It should be implemented in the GlobalAlignment task rather than the SimpleGlobalAlignment subclass.